main.go

```diff
@@ -249,12 +295,45 @@ func (p *UnixProcess) Refresh() error {
 	// Move past the image name and start parsing the rest
 	data = data[binStart+binEnd+2:]
 	_, err = fmt.Sscanf(data,
-		"%c %d %d %d",
+		"%c %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d",
 		&p.state,
 		&p.ppid,
 		&p.pgrp,
-		&p.sid)
+		&p.sid,
+		&p.tty_nr,
+		&p.tpgit,
+		&p.flags,
+		&p.minflt,
+		&p.cminflt,
+		&p.utime,
+		&p.stime,
+		&p.cutime,
+		&p.cstime,
+		&p.nice,
+		&p.num_threads,
+		&p.itrealvalue,
+		&p.starttime,
+		&p.vsize,
+		&p.rss,
+		&p.rsslim,
+		&p.startcode,
+		&p.endcode,
+		&p.startstack,
+		&p.kstkesp,
+		&p.kstkeip,
+		&p.signal,
+		&p.blocked)
+
+	return err
+}
 
+func (p *UnixProcess) RefreshCmdline() error {
+	cmdlinePath := fmt.Sprintf("/proc/%d/cmdline", p.pid)
```

Comment:
it's it easier to check for existence of `cmdline` and return blank cmdline if it does not exist. Anyway, I having hard time to trace what happens now if this file does not exist, please double check that

Reply:
Per http://man7.org/linux/man-pages/man5/proc.5.html the file always exists, but can be empty for zombies.